python/GafferUI/NodeGraph.py

```diff
@@ -285,9 +294,11 @@ def __keyPress( self, widget, event ) :
 		elif event.key == "Down" :
 			selection = self.scriptNode().selection()
 			if selection.size() :
-				needsModifiers = not isinstance( selection[0], ( Gaffer.Reference, Gaffer.Box ) )
-				haveModifiers = bool( event.modifiers & ( event.modifiers.Shift | event.modifiers.Control ) )
-				if needsModifiers == haveModifiers :
+				needsModifiers = not Gaffer.Metadata.value( selection[0], "nodeGraph:childrenViewable" )
+				if (
+					( needsModifiers and event.modifiers == event.modifiers.Shift | event.modifiers.Control ) or
```

Comment:
I _think_ everyone here already thought it required shift+ctrl, so I guess while this is more demanding now, its not gonna stop anyone at IE from diving inside. I could be wrong though.